regression/cbmc/array-cell-sensitivity9/test.desc

```diff
@@ -0,0 +1,32 @@
+CORE
+test.c
+--show-vcc
```

Comment:
It seems that all the tests only exercise `--show-vcc` - I think we should also be running the full flow (in addition to `--show-vcc`) to make sure we actually get the right results!